pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/tasks/action/PipWheelActionTest.groovy

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.tasks.action
+
+import com.linkedin.gradle.python.exception.PipExecutionException
+import com.linkedin.gradle.python.extension.PythonDetailsTestDouble
+import com.linkedin.gradle.python.extension.VirtualEnvironment
+import com.linkedin.gradle.python.extension.WheelExtension
+import com.linkedin.gradle.python.tasks.action.pip.PipActionHelpers
+import com.linkedin.gradle.python.tasks.action.pip.PipWheelAction
+import com.linkedin.gradle.python.tasks.exec.ExternalExecTestDouble
+import com.linkedin.gradle.python.util.DefaultEnvironmentMerger
+import com.linkedin.gradle.python.util.PackageSettings
+import com.linkedin.gradle.python.wheel.EmptyWheelCache
+import org.gradle.process.ExecSpec
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+
+import static com.linkedin.gradle.python.tasks.action.pip.PipActionHelpers.packageInGradleCache
+
+
+class PipWheelActionTest extends Specification {
+
+    @Rule
+    TemporaryFolder temporaryFolder
+
+    def "wheel install uses environment"() {
```

Comment:
Please, remove the word "install" here and in other tests below.
The command given to pip in these executions is "wheel" and that word is enough to describe that this command does the given described action. The "install" was needed above in the "pip install" tests.